### ISSUE TITLE
Add a "submitted" status to job history

### ISSUE THREAD

**Issue by bradtaylor** [DataBiosphere/terra-ui#815]
One of our users asked us to distinguish between jobs that have been submitted from jobs that are running. This is important information for her as she monitors to see if anything is going wrong.

To make this distinction, we can add another status, "submitted", to the set of (running, succeeded, failed) that we use now. We need an icon for this. A clock face perhaps?

Rawls submits work to cromwell. There is a queue in front of it, and jobs in cromwell may linger in a submitted state before starting.

**Comment by elizzhao** [comment]
Is this a duplicate of #1479 ?

**Comment by panentheos** [comment]
Closed as duplicate.
